app/posts/apply-for-teacher-training/2020-10-08-getting-references-before-submitting.md

```diff
@@ -0,0 +1,301 @@
+tags:
+- AN024
+- AN023
+- AN013
+- AN010
+---
+
+Obtaining references is a pain point for many candidates. Identifying appropriate referees can be difficult, especially for recent graduates and candidates who have been out of the workplace. Once they have been identified, ensuring they respond, can again be challenging.
+
+## Our initial approach
+
+For the 2019/20 cycle, we deviated from UCAS’ reference process. Instead of preventing candidates from submitting their application until they had 2 complete references, we allowed them to complete their application, only contacting referees after submission.
+
+By the time the 2019/202 recruitment cycle had closed:
+
+* 687 candidates had received references.
+* The average time for getting a reference was 6 days
+* 97.8% of candidates that received references got them within 30 days
+* 86 candidates were still waiting for references. Of these, 11 had waited longer than 30 days.
+
+Our initial design intended to make the process easier for candidates, but instead made it more opaque; many candidates believed applications were sent directly to the provider when they pressed ‘Submit’, not put in a holding state while we waited for their references to come back.
+
+Once candidates understood that any delay in getting a reference would prevent an application from reaching a provider, the process became frustrating. Candidates felt powerless while waiting on a third-party, unable to control their own destiny.
+
+Further issues with this process became evident towards the end of the recruitment cycle:
+
+* 89 courses were full by the time references were returned for 60 candidates (as of early July 2020). We tried to mitigate this situation by [giving candidates the option to replace a course choice](/apply-for-teacher-training/course-full-after-submitting-initial-approach/), but no candidate used this feature.
+
+* The Apply deadline was moved forwards 2 weeks to account for the average delay in getting a reference. This in turn required additional business rules and communications with candidates, and introduced additional complexity and confusion given the UCAS journey maintained the original deadline.
+
+## Decoupling references from the application process
+
+To address these issues, we considered designing a system which would allow us to ‘decouple’ references from the point of submission.
+
+What if candidates could request references before, or after submitting an application, with references attached to an application as they arrived? Maybe providers could be made the condition of an offer?
+
+However, given prior research with providers, we know there are some constraints with this approach:
+
+* Providers want to see 2 references when they review an application
+
+* While we are dual-running with UCAS we can’t deviate too far from the existing business rules. This is to maintain fairness; candidates applying via UCAS would have references, while candidates coming via our service may not
```

Comment:
That renders oddly, does not have the standard list item spacing:

![Screen Shot 2020-10-08 at 17 34 27](https://user-images.githubusercontent.com/319055/95487673-89637480-098c-11eb-827c-86df1414840e.png)